tests/test_app.py

```diff
@@ -76,7 +76,6 @@ def test_app_configure_with_platforms(mocked_engine):
         token='should-be-a-valid-token'
     )
     mocked_engine['instance'].configure.assert_called_with()
-    mocked_engine['instance'].tasks.assert_called_with()
```

Comment:
the `id` and `first_name` are both not optional, so, it'll always come from the Telegram API.

Reply:
Wrong comment? haha